src/content/docs/extensions/httpfs.mdx

```diff
@@ -1,28 +1,33 @@
 ---
 title: HTTP file system extension
+description: "The HTTPFS extension allows you to read from files hosted on remote file systems"
 ---
 
 The `httpfs` extension extends the Kuzu file system by allowing reading from/writing to files hosted on
-remote file systems. The following remote file systems are supported:
+remote file systems. The following extensions are provided via HTTPFS, each supporting a different remote file system:
 
-- Plain HTTP(S)
-- Object storage via the AWS S3 API
-- Object storage via the Google Cloud Storage (GCS) API
+- This extension: Plain HTTP(S) file systems
+- [S3 extension](/extensions/s3): Object storage via the AWS S3 API
+- [GCS extension](/extensions/gcs): Object storage via the Google Cloud Storage (GCS) API
```

Comment:
It's the same extension, so would be misleading to say these are different _extensions_. Maybe "the following features"?
```suggestion
-  Plain HTTP(S) file systems
- Object storage via the AWS S3 API. See [S3](/extensions/s3). #FIXME link text
- Object storage via the Google Cloud Storage (GCS) API. See [GCS](/extensions/gcs). #FIXME link text
```

Reply:
yeah, saying their features sounds more reasonable. Will fix.